docs/common/craft-parts/reference/plugins/rust_plugin.rst

```diff
@@ -174,8 +174,8 @@ To get even better performance, you might want to follow the tips below.
 
 * (Advanced) Perform cross-language LTO. This requires installing the correct version of LLVM/Clang and setting the right environment variables.
   You must know which LLVM version of your selected Rust toolchain is using.
-  For example, Rust 1.71 uses LLVM 16 because you can see it bundles a ``libLLVM-16-rust-1.71.1-stable.so`` file in the ``lib`` directory.
-  In this case, you want to install ``clang-16`` and ``lld-16`` from the Ubuntu archive.
+  For example, Rust 1.81 uses LLVM 18 because you can see it bundles a ``libLLVM.so.18.1-rust-1.81.0-stable`` file in the ``lib`` directory.
```

Comment:
Is there a document somewhere that lists the correct LLVM versions for rust versions?

Reply:
I couldn't find anything definitive in a cursory search. Their doc on upgrading LLVM points to GH PRs, so they might be the most reliable way to check. Said PRs at least have  consistent naming: https://github.com/rust-lang/rust/pulls?q=is%3Apr+is%3Aclosed+in%3Atitle+%22Update+to+LLVM%22+

Reply:
> Is there a document somewhere that lists the correct LLVM versions for rust versions?

You might also do `rustc -vV` on newer Rust versions, which will print out something like:

```
rustc 1.81.0 (eeb90cda1 2024-09-04)
binary: rustc
commit-hash: eeb90cda1969383f56a2637cbd3037bdf598841c
commit-date: 2024-09-04
host: x86_64-unknown-linux-gnu
release: 1.81.0
LLVM version: 18.1.7
```

Reply:
@liushuyu could you include that tip in the doc for users?